types/product-drop.d.ts

```diff
@@ -1,3 +1,4 @@
+import { Response } from "node-fetch";

```

Comment:
This sounds a bit off. Can you provide an example of the actual response? Returning nothing (void) is an option.

Reply:
We are returning what pusher trigger returns: https://github.com/pusher/pusher-http-node/blob/85c8c14d841d12fd41fdeefe0f3a0e9ae64b5595/index.d.ts#L14

Reply:
Ok let's abstract that away from clients and avoid leaking implementation details (Pusher vs Amazon Kinesis vs whatever service we use for events). Alternatives:
1 - Return nothing
2 - Return some kind of normalized structure (adapter pattern)

In this case 1 seems enough since the return won't be used for anything.